tools/pd-ctl/README.md

```diff
@@ -645,16 +644,16 @@ Usage:
 >> store label 1 zone cn               // Set the value of the label with the "zone" key to "cn" for the store with the store id of 1
 >> store weight 1 5 10                 // Set the leader weight to 5 and region weight to 10 for the store with the store id of 1
 >> store remove-tombstone              // Remove stores that are in tombstone state
->> store limit                         // Show limits of adding region operation for all stores
->> store limit region-add              // Show limits of adding region operation for all stores
->> store limit region-remove           // Show limits of removing region operation for all stores
+>> store limit                         // Show limits of adding peer operation for all stores
+>> store limit add-peer                // Show limits of adding peer operation for all stores
```

Comment:
we need a warning like:
```
`region-add` is deprecated, please use `add-peer`
```
if the user uses old command.
and I more likely use old command also works.